libkbfs/disk_journal.go

```diff
@@ -36,8 +37,6 @@ import (
 //
 // TODO: Do all high-level operations atomically on the file-system
 // level.
-//
-// TODO: Make IO ops cancellable.
```

Comment:
I removed it because I was thinking about it and became skeptical that the complexity would be worth it. But I put it back, as we can always make that determination when we try to implement it (or remove it if/when we move to a database).